src/components/controller/loader.ts

```diff
@@ -23,18 +22,18 @@ class Loader {
         return res;
     }
 
-    makeUrl(options, endpoint) {
+    makeUrl(options = {}, endpoint: string) {
         const urlOptions = { ...this.options, ...options };
         let url = `${this.baseLink}${endpoint}?`;
 
-        Object.keys(urlOptions).forEach((key) => {
-            url += `${key}=${urlOptions[key]}&`;
+        Object.entries(urlOptions).forEach((value: [string, unknown], index?: number, array?: [string, unknown][]) => {
+            url += `${value[0]}=${value[1]}&`;
         });
```

Comment:
```suggestion
        Object.keys(urlOptions).forEach((value: string, index?: number) => {
            url += `${value}=${urlOptions[value]}&`;
        });
```
need check, but I hope must work without unknown

Reply:
throws an error then